documentation/jetty-documentation/src/main/asciidoc/programming-guide/migration/migration.adoc

```diff
@@ -0,0 +1,130 @@
+//
+// ========================================================================
+// Copyright (c) 1995-2021 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+[appendix]
+[[_migration]]
```

Comment:
Please fix the id as the other documents.
This is the programming guide so the id should be `[[pg-migration]]`

Reply:
I have redone the links for all the headers in the migration guide. Can you check I have done this right, I have prefixed every id with the parent id.